bin/launch_ball.py

```diff
@@ -2,16 +2,19 @@
 
 """
 Connects to ball launcher beepy and send a ball.
-The ball launcher server should be running on the 
+The ball launcher server should be running on the
 raspberry pi of the ball launcher.
 """
 
-import time, typing
-from ball_launcher_beepy import BallLauncher
+import typing
+
+from ball_launcher_beepy.ball_launcher_client import BallLauncherClient
 
 
 class _BallLauncherConfig:
-    def __init__(self):
+    """Stores launch configuration of launch dialog."""
+
+    def __init__(self) -> None:
```

Comment:
(I know not your fault) can you create a global variable IP and PORT after the import ? makes it easier to find/correct the hard coded IP and PORT